po/de.po

```diff
@@ -284,7 +284,7 @@ msgstr "Benachrichtigungen sind deaktiviert"
 
 #: src/Common/Main.vala:537
 msgid "Can not reach site"
-msgstr "Die site kann nicht erreicht werden"
+msgstr "Die Webseite kann nicht erreicht werden"
```

Comment:
Don't capitalize website, unless there is some rule of German grammar that says it should be because it is the main object or something. Otherwise, it's not a title or a heading or a name, it's just a word in a message in this context.